docs/src/gui/qtvcp-code-snippets.adoc

```diff
@@ -69,30 +81,31 @@ self.SETTINGS_.setValue('myFloat', self.float_value)
 self.SETTINGS_.endGroup()
 ----
 
-== Add a basic style editor
-
-Being able to edit a style on a running screen is convienant.
+== Add A Basic Style Editor
```

Comment:
```suggestion
== Add a Basic Style Editor
```

And some further, I didn't mark all.

Reply:
Could we leave it titles as they are for this PR and wait a later one once we decide on titles capitalization ?
Or can we decide now and I fix them all right away.

Again, personnal choice is to adopt "Sentence formatting" with only first char in capital, except other rightly capitalized words of course.

Reply:
Yes, just marked was catched my eye...